util/maven/generate_poms.py

```diff
@@ -58,6 +58,29 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//java/dagger:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
+            <dependency>
+              <groupId>%s</groupId>
```

Comment:
javax.inject comes transitively from dagger core so it's not necessary to list it here.

We cannot use a `deps` on the `java_library` as that requires having `srcs` too (this is why in a previous version I used `exports` and had to change `generate_poms.py`).